policy_tests/install_kernels.py

```diff
@@ -18,17 +18,19 @@ def test_install_kernel(policy, debug):
         install_path = "-".join([install_path, "debug"])
     if not os.path.isdir(install_path):
         os.makedirs(install_path)
-    
+
+    val_name = "librv32-renode-validator.so"
```

Comment:
Does QEMU/policy-engine not change the name of the validator to start with `rv64`?

Reply:
The policy engine does, but the scripts copy it to this name, since it is hard-coded into the QEMU config scripts.  (The name's already a misnomer since it's not for renode.)

Reply:
My PR will fix the renode issue (finally).
Perhaps the way to go for the name here is to leave it devoid of architecture references. Like `policy-validator.so`

Reply:
I think using the same name would have a similar issue because then we would have to use a different search path based on which QEMU was being used.

Regardless, I now remember why I took this shortcut, because I was sick of fighting with the QEMU build system.  Resuming the struggle I believe I have a way to handle it:
https://github.com/draperlaboratory/hope-qemu/tree/64_bit_linking